src/main/java/com/timeplus/kafkaconnect/TimeplusSinkTask.java

```diff
@@ -169,17 +173,20 @@ void createStream(String event) {
                 .post(body)
                 .build();
 
-        Response response;
+        Response response = null;
         try {
             response = client.newCall(request).execute();
-            if (response.isSuccessful()) {
+            if (response != null && response.isSuccessful()) {
```

Comment:
Just curious, can `response` be `null` here? I assume `response` can only be `null` when exceptions were thrown?